public/UI.js

```diff
@@ -54,7 +54,8 @@ class UIController {
             while (statusBody.firstChild) {
                 statusBody.removeChild(statusBody.firstChild);
             }
-            let lastUpdate = data["last_update"];
+            let fullDateFormat = d3.timeFormat("%d %b %Y %H:%M:%S %p"),
```

Comment:
Hi @DanielMwendwa I meant to make the graph.js last updated to be consistent with coding progress Last updated: 2020-02-13 06:25:04